### PR TITLE
Platform (OS name, Version) common interface.

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src_app/test_scenes/sceneModelAnimated.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
 import React from 'react';
+import { Factory } from 'magic-script-components';
 
 class SceneModelAnimated extends React.Component {
   render () {
+    const path = (Platform.OS === 'Android') ? require('../../resources/hedra_anim.glb') : require('../../resources/hedra_anim.glb')
```
Comment (pleszkiewicz) [inline src_app/test_scenes/sceneModelAnimated.js]: Please do not change `sceneModelAnimated.js` in this PR. There is already another branch with changes to models scenes waiting for `Platform` support.

File: src_app/catalog_app.js [hunk 2]
```diff
@@ -93,6 +93,8 @@ class CatalogApp extends React.Component {
 
     const initialIndex = this.scenes.findIndex((item) => item.name == 'Characters');
     // const initialIndex = this.scenes.length - 6;
+    console.log('Platform: ', Platform.OS);
```
Comment (pleszkiewicz) [inline src_app/catalog_app.js]: Can you put it in one line?
```
console.log(`Runs on ${Platform.OS} v.${Platform.Version}`);
```

**Review by pleszkiewicz — APPROVED** [review]
👍

**Review by kpiascik — APPROVED** [review]
This change should be made to the CLI template

**Pull request merged**